### DISCUSSION TITLE
Towny Chest Shops

### DISCUSSION THREAD

**Discussion by tvgblade** [TownyAdvanced/Towny#5868] [Q&A]
I am unable to create chest shops in the shop plots, is there a specific config option that is needed, and before it is asked yes i have placed a chest down and right clicked it with an item

**Comment by LlmDl** [comment]
What is the plugin adding chest shops?

**Comment by tvgblade** [comment]
I am unsure if it's the plug-ins as every town wiki i find mentions chest
shops and that they are available to use when u have a shop plot

**Reply by LlmDl** [reply]
You might be talking about QuickShop or ChestShop, both can be limited to Towny's Shop Plots.

You might not be able to create a shop, because you have no Shop plugin installed yet.

**Comment by tvgblade** [comment]
Ahh okay thank you so much for your help!!!
